frontend/src/pages/ApiParticulier/index.js

```diff
@@ -337,12 +337,13 @@ const editorList = [
   { name: 'Qiis', siret: '52834311400021' },
   { name: 'Aiga', siret: '39825361700045' },
   { name: 'Teamnet', siret: '33922000600078' },
-  { name: 'JVS', siret: '32855218700069' },
+  { name: 'JVS-MAIRISTEM', siret: '32855218700069' },
```

Comment:
la casse ne correpond pas à ce qui est affiché sur le site https://www.jvs-mairistem.fr/
```suggestion
  { name: 'JVS-Mairistem', siret: '32855218700069' },
```